AISKU/src/ApplicationInsightsDeprecated.ts

```diff
@@ -1,11 +1,11 @@
 import {
```

Comment:
Because the Minify script automatically "adds" the name replacements before compiling and then removes them after compiling it also "updates" the imports of every file by

- Removed all unused imports (unless it contains comments -- it leaves these alone)
- Sorts the imported values, with "default" imports `import xxx from yyy;` at the top
- Sorts the imports with all "@" imports at the top